config/eigendaflags/v2/cli.go

```diff
@@ -184,34 +200,38 @@ func ReadConfig(ctx *cli.Context) common.V2ClientConfig {
 }
 
 func readPayloadClientConfig(ctx *cli.Context) v2_clients.PayloadClientConfig {
-	polyForm := codecs.PolynomialFormCoeff
+	polyForm := codecs.PolynomialFormEval
 
 	// if point evaluation mode is disabled then blob is treated as evaluations and
```

Comment:
this comment needs to be updated as well:

"...then blob is treated as evaluations..." -> "...then blob is treated as coefficients..."

Reply:
[8f8dcd2](https://github.com/Layr-Labs/eigenda-proxy/pull/298/commits/8f8dcd210143af210b04299af9ceaf6569716eb5)